java/compaction/compaction-job-execution/src/test/java/sleeper/compaction/jobexecution/testutils/CompactSortedFilesTestUtils.java

```diff
@@ -64,14 +63,9 @@ public static void assertReadyForGC(StateStore dynamoStateStore, FileInfo... fil
     }
 
     public static void assertReadyForGC(StateStore stateStore, List<FileInfo> files) throws StateStoreException {
-        assertThat(stateStore.getReadyForGCFiles()).toIterable()
-                .extracting(
-                        FileInfo::getFilename,
-                        FileInfo::getPartitionId,
-                        FileInfo::getFileStatus)
-                .containsExactlyInAnyOrder(files.stream()
-                        .map(file -> tuple(file.getFilename(), file.getPartitionId(),
-                                FileInfo.FileStatus.READY_FOR_GARBAGE_COLLECTION))
-                        .toArray(Tuple[]::new));
+        assertThat(stateStore.getReadyForGCFilenamesBefore(Instant.ofEpochMilli(Long.MAX_VALUE)))
+                .containsExactlyInAnyOrderElementsOf(files.stream()
+                        .map(FileInfo::getFilename)
+                        .collect(Collectors.toSet()));
```

Comment:
It seems misleading to get rid of duplicates before we do the assertion. It might be easier and clearer to do the assertion directly in the tests now that it's just asserting on the filenames.